packages/docs/src/styles/components/navigation.scss

```diff
@@ -11,8 +11,13 @@
     gap: $spacer-2;
     padding: $spacer-2;
 
-    button {
+    > button {
       color: currentColor;
+
+      &:hover,
+      &:focus:hover {
+        color: currentColor;
```

Comment:
noticed the 🍔 icon is illegible when `:focus:hover` is applied and fixed that

Reply:
Oh yeah, I never noticed that because I must have always tested it in touch-device mode